apps/src/templates/rubrics/AiAssessmentBox.jsx

```diff
@@ -51,30 +49,10 @@ export default function AiAssessmentBox({
           <BodyThreeText>
             <StrongText>{studentAchievment()}</StrongText>
           </BodyThreeText>
-          {aiConfidence && (
-            <div>
-              <EmText>{aiConfidenceText()}</EmText>
-              <span data-tip data-for="info-tip">
-                <FontAwesome icon="info-circle" className={style.infoTipIcon} />
-              </span>
-              <ReactTooltip id="info-tip" effect="solid">
-                PLACEHOLDER TEXT FOR INFO TIP
```

Comment:
It seems like since we have integrated the confidence scores, we SHOULD have this tool tip here.  Obviously I will defer to Tess/Molly, but as a teacher, I would be frustrated if AI has a "low confidence" but I don't actually know what that means or what I should do about that vs a "high confidence".   That information isn't really useful without context.   If we aren't able to come up with text that provides that context I might argue that we shouldn't be providing confidence scores to teachers.  Also, it is possible that you and Tess already had this conversation and landed on removing this text, so again, I defer to you two.

Reply:
Good question! I'm going to move the question over to Slack